packages/openneuro-server/graphql/resolvers/snapshots.js

```diff
@@ -141,8 +141,20 @@ export const updateSnapshotFileUrls = (obj, { fileUrls }) => {
 }
 
 const Snapshot = {
-  analytics: snapshot => analytics(snapshot),
-  issues: snapshot => snapshotIssues(snapshot),
+  analytics: () => analytics,
+  issues: snapshotIssues,
+  description: snapshot =>
+    description(snapshot, {
+      datasetId: snapshot.datasetId,
+      tag: snapshot.tag,
+    }),
+  readme: snapshot =>
+    readme(snapshot, {
+      datasetId: snapshot.datasetId,
+      revision: snapshot.hexsha,
+    }),
+  files: (snapshot, { prefix }) =>
+    getFiles(snapshot.datasetId, snapshot.hexsha).then(filterFiles(prefix)),
```

Comment:
I'm getting nulls for files, readme, and description when fetching snapshots from dataset pages. It looks like getFiles requests are being made to url: "datalad:9877/datasets/undefined/snapshots/null/files".